pimd/pim_iface.c

```diff
@@ -1536,6 +1536,12 @@ void pim_if_create_pimreg(struct pim_instance *pim)
 			snprintf(pimreg_name, sizeof(pimreg_name), PIMREG "%u",
 				 pim->vrf->data.l.table_id);
 
+		if (pim_reg_sock_bind(pim)) {
+			zlog_warn(
```

Comment:
Do we print zlog_warn() twice if pim_socket_bind() fails?

Reply:
Agree, I will remove the one in the caller